New version: Stipple v0.31.0

JuliaRegistrator opened the pull request:
- Registering package: Stipple
- Repository: https://github.com/GenieFramework/Stipple.jl
- Created by: @hhaensel
- Version: v0.31.0
- Commit: 8762bdab74ba77204bdc619621d428812c637cdd
- Reviewed by: @hhaensel
- Reference: https://github.com/GenieFramework/Stipple.jl/commit/8762bdab74ba77204bdc619621d428812c637cdd#commitcomment-151955215
- Description: The reactive UI library for interactive data applications with pure Julia.
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes
- new theme and handling of user themes

## News

- referring to variables in the `@app` declaration, e.g. `@in y = x + 1`
- multiple models on one page
- debugging of model events with `Stipple.debug()`
- add `throttle` in analogy to `debounce`
- `@handler` macro to define app handlers outside of `@app`
- handler merging in mixins
- accept Inf and NaN in floats (full support with next release of JSON3)
- routehandlers `pre` / `post` for `@page` 
- precompilation by `@stipple_precompile`
### Fixes / Improvements
- reconnection after network interruption, e.g. energy save mode
- `@async` calls in `@onchange` sections
- remove state-changing evals from ReactiveTools macros
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd611c749ef5ad346264be05b7307d1de4584b82fdd554b9b531e22a3afa75a29627362bc3e3cdb43096b19dc3e08be68fe443fb91c06525e331b05a388bf7b263f82077447581d2e41c0e29da969d022ed1b198799912a15c8da62620ab12a4ef2d0df41f2d96ebe4446b9a5e4ffb57103f8382424fcc98f275f6b363ac45aed62afa3c01432f876af375fdbeb74216df18f012512518276a24c524d80ad1f6cdca38a3a5c2178db122a94b86cd75e0a1f -->

hhaensel commented:
@JuliaRegistrator register

Release notes:

## Breaking changes
- new theme and user theme handling

## New Features
- referring to variables in the `@app` declaration, e.g. `@in y = x + 1`
- multiple models on one page
- debugging of model events with `Stipple.debug()`
- add `throttle` in analogy to `debounce`
- `@handler` macro to define app handlers outside of `@app`
- handler merging in mixins
- accept Inf and NaN in floats (full support with next release of JSON3)
- routehandlers `pre` / `post` for `@page` 
- precompilation by `@stipple_precompile`
### Fixes / Improvements
- reconnection after network interruption, e.g. energy save mode
- `@async` calls in `@onchange` sections
- remove state-changing evals from ReactiveTools macros

JuliaRegistrator committed:
New version: Stipple v0.31.0

UUID: 4acbeb90-81a0-11ea-1966-bdaff8155998
Repo: https://github.com/GenieFramework/Stipple.jl.git
Tree: e4c66f9074aa0897f7d5922b33c0b03bcc48197a

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
